### DISCUSSION TITLE
File Upload Functionality in artillery

### DISCUSSION THREAD

**Discussion by JayasimhaNagaraj** [artilleryio/artillery#971] [Artillery]
Hi All,

Was stuck in a file upload scenario. Would like to know if there is an option for achieving file upload scenarios via artillery.
There was one documentation which mentioned the support for multipart/form-data form uploads. But the below documentation doesn’t exist.
https://artillery.io/docs/pro/file-uploads

Any pointers around this would be helpful.

Thank you

**Comment by hassy** [comment]
hi @JayasimhaNagaraj👋 Perfect timing for this question! File uploads *right now* are only available in Artillery Pro, **however** we're about to make them available to everyone in the next release of Artillery Core. Stay tuned 📻 :)

**Reply by JayasimhaNagaraj** [reply]
Hi Hassy,

Thanks for the update. That's a great news. However, was curious to know if there is any tentative timeline for the next release ?

**Reply by hassy** [reply]
As soon as https://github.com/artilleryio/artillery/pull/974 is merged :)

**Reply by kshamajk** [reply]
Hi Hassy,

 Wanted to know if there are any restrictions on type of file to be uploaded ? Considering our projects, we have different scenarios, where we upload .pdf files, excel/csv files, .png files etc.  Are these supported ?

Thanks
